### PR TITLE
KEYCLOAK-19143 Split note for broker and SP SAML request ID

### PR DESCRIPTION
**Pull request by hmlnarik**
Follow-up on #8110

### CODE REVIEW

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/saml/BrokerTest.java [hunk 1]
```diff
@@ -323,6 +367,8 @@ public void testAnyNamespacePreservedInContext() throws IOException {
                 .build()
               .assertResponse(org.keycloak.testsuite.util.Matchers.statusCodeIsHC(Status.OK))
               .execute();
+        } finally {
+            clearUsers(realm);
```
Comment (mhajas) [inline testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/saml/BrokerTest.java:371]: Why is this change necessary? Does this test create some users that needs to be cleared?

**Review by mhajas — DISMISSED** [review]
Thank you for the PR @hmlnarik. The change looks correct. I added one question into the tests but it looks good so I am approving.

File: testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/saml/BrokerTest.java (hunk 1 above)
Comment (hmlnarik) [reply testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/saml/BrokerTest.java]: Thanks, good catch :). Indeed it's not, omitted from a working state.

**Review by mhajas — APPROVED** [review]
Thanks for the fix, approving again.
